### PR TITLE
Merge 4.5.2 into 4.6.0

### PR DESCRIPTION
**Pull request by damarisg**
|Related issue|
|-------------|
|        https://github.com/wazuh/wazuh-qa/issues/4344     |

## Description

Scheduled upward merges for numbered branches - 4.5.2 into 4.6.0

### CODE REVIEW

File: version.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
     "version": "4.6.0",
     "revision": "40600"
-}
+}
```
Comment (BelenValdivia) [inline version.json:4]: Add end line

**Review by BelenValdivia — CHANGES_REQUESTED** [review]

File: version.json (hunk 1 above)
Comment (damarisg) [reply version.json]: Done https://github.com/wazuh/wazuh-qa/pull/4348/commits/5130d205c46d3a5bf1e8d70f447a543994870f5f

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,14 +4,18 @@ All notable changes to this project will be documented in this file.
 
 ## [4.6.0] - TBD
 
-Wazuh commit: TBD \
-Release report: TBD
-
```
Comment (rauldpm) [inline CHANGELOG.md:9]: Shouldn't this be maintained?

**Review by rauldpm — CHANGES_REQUESTED** [review]

**Review by QU3B1M — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (damarisg) [reply CHANGELOG.md]: Done: https://github.com/wazuh/wazuh-qa/pull/4348/commits/6bc115abf08895aa5419cfd72d52972d05246d60

**Review by BelenValdivia — APPROVED** [review]

**Review by rauldpm — APPROVED** [review]

**Comment by damarisg** [comment]
I will merge because check failures will be removed. (https://github.com/wazuh/wazuh-qa/issues/4291)

**Pull request merged**